src/Delegates/handlesIndex.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace MeiliSearch\Delegates;
+
+use MeiliSearch\Endpoints\Indexes;
+
+trait handlesIndex
```

Comment:
Must be uppercase on `h`

Reply:
Actually I was thinking that the camelCased trait names would subtly indicate that its a trait instead of a class being used, If my memory is correct somewhere I have seen this practice (possibly laravel). So I had followed them in my projects as well. 

But when I went back and checked they are also using CapitalCamelCase traits. So I have renamed all the traits to be CapitalCamelCased.